packages/training/deploy/docker/scripts/benchmark-entrypoint.sh

```
@@ -0,0 +1,374 @@
+EFFECTIVE_MODEL="$BASE_MODEL"
+if [ -n "$MODEL_PATH" ] && [ -d "$MODEL_PATH" ]; then
+    # Check if it's a LoRA adapter (has adapter_config.json) or a full model
+    if [ -f "$MODEL_PATH/adapter_config.json" ]; then
+        log_info "Loading LoRA adapter: $MODEL_PATH"
+        VLLM_CMD="$VLLM_CMD --enable-lora"
+        VLLM_CMD="$VLLM_CMD --lora-modules trained-adapter=$MODEL_PATH"
+        ADAPTER_NAME="trained-adapter"
+    else
+        log_info "Loading merged model: $MODEL_PATH"
+        # Override the base model with the merged model path
+        VLLM_CMD=$(echo "$VLLM_CMD" | sed "s|--model $BASE_MODEL|--model $MODEL_PATH|")
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)